src/components/ActionList/ActionList.tsx

```diff
@@ -34,15 +34,15 @@ export default function ActionList({
   // see https://github.com/Microsoft/TypeScript/issues/28768
   const Element: any = hasMultipleSections ? 'ul' : 'div';
   const sectionMarkup = finalSections.map((section, index) => {
-    return (
+    return section.items.length > 0 ? (
       <Section
         key={section.title || index}
         section={section}
-        onActionAnyItem={onActionAnyItem}
         hasMultipleSections={hasMultipleSections}
         actionRole={actionRole}
+        onActionAnyItem={onActionAnyItem}
       />
-    );
+    ) : null;
```

Comment:
Why was this change necessary?

Reply:
Nothing within `items: ActionListItemDescriptor[];` is a required prop, so `section.items` could equal `[]`, which meant returning a useless result.

Reply:
related: https://github.com/Shopify/polaris-react/issues/1676